### PR TITLE
User create new recipe

### PR DESCRIPTION
**Pull request by moscc**
### What we did?
- Created a form for adding new recipe
- Added new and create actions in controller
- Added new and create routes for recipes

### What we have learned?
- Create forms

### CODE REVIEW

File: features/user_create_new_recipe.feature [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  Background:
+    Given We have the following user:
+      | email          | password |
+      | maran@test.com | 12345678 |
+
+    And I visit the login page
+    And I fill in "Email" with "maran@test.com"
+    And I fill in "Password" with "12345678"
+    And I click "Log in"
+    And I visit the site
```
Comment (tochman) [inline features/user_create_new_recipe.feature]: Please ide Warren test helpers to login in scenarios where you need a current_user instance

File: app/controllers/recipes_controller.rb [hunk 2]
```diff
@@ -1,4 +1,17 @@
 class RecipesController < ApplicationController
+  def new
+    @recipe = Recipe.new
+  end
+
+  def create
+    @recipe = Recipe.new(product_params)
+    if @recipe.save
+      redirect_to root_path
```
Comment (tochman) [inline app/controllers/recipes_controller.rb]: No flash message?

File: features/user_create_new_recipe.feature (hunk 1 above)
Comment (moscc) [reply features/user_create_new_recipe.feature]: Ok will do

File: app/controllers/recipes_controller.rb (hunk 2 above)
Comment (moscc) [reply app/controllers/recipes_controller.rb]: Going for mvp, will add it.

Comment (moscc) [reply app/controllers/recipes_controller.rb]: Done

File: features/user_create_new_recipe.feature (hunk 1 above)
Comment (moscc) [reply features/user_create_new_recipe.feature]: Done

**Review by tochman — APPROVED** [review]
Looks good to me

**Pull request merged**